build1.sh

```diff
@@ -3,15 +3,36 @@
 set -e
 set -x
 
-cmake \
+SRCDIR=`pwd`
+CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH_LPYTHON;$CONDA_PREFIX"
+
+# if we are in a pixi project, use the pixi "host" environment for the dependencies
+if [[ ! -z $PIXI_PROJECT_ROOT ]]; then
+   CMAKE_PREFIX_PATH=$PIXI_PROJECT_ROOT/.pixi/envs/host/
+   OUTOFTREE=false
+
+   if [[ ! -f src/libasr/asr.h ]]; then
+       pixi run build_0
+   fi
+fi
+
+# if the outoftree env var is set to true, create a build directory
+if [[ "$OUTOFTREE" == "true" ]]; then
+    mkdir -p build
+    cd build
+fi
```

Comment:
Is this needed for `pixi build`? If so, why not just move this logic to `pixi.toml` itself? That way this simple script can stay build system independent.